src/actions.sol

```diff
@@ -92,6 +92,9 @@ contract Actions is DSNote {
     }
 
     function repay(ShelfLike shelf, ERC20Like erc20, uint loan, uint amount) public {
+        uint debt = PileLike(shelf.pile()).debt(loan);
```

Comment:
If a local variable is only used once it is cheaper to just replace it with the direct call.